program/c/makefile

```diff
@@ -7,6 +7,6 @@ cpyth:
 	bash -c "ar rcs target/libcpyth.a target/**/*.o"
 cpythtest:
 #   Compile C code to system architecture for use by rust's cargo test
-	$(CC) -c ./src/oracle/for_cargo_test/cpyth_test.c -o ./target/cpyth_test.o
+	gcc -c ./src/oracle/for_cargo_test/cpyth_test.c -o ./target/cpyth_test.o
 #   Bundle C code compiled to system architecture for use by rust's cargo test
```

Comment:
Had to change this back... maybe $CC in the context of this makefile has the bpf flags?

Reply:
hmmm ok weird. i'm not really sure how this whole makefile process works, tbh.